### DISCUSSION TITLE
[FIXED] ⚠️ Avoid macOS Big Sur system update to XCode CMD line tools 12.5 ⚠️

### DISCUSSION THREAD

**Discussion by dschwen** [idaholab/moose#18134] [News]
<img width="434" alt="Screen Shot 2021-06-21 at 10 22 58 AM" src="https://user-images.githubusercontent.com/202302/122839572-85c31400-d2b5-11eb-9a3b-6630041b0131.png">

The current macOS update will bump your XCode CMD line tools version to 12.5, which is unsupported by MOOSE at this time. Avoid this, or you will have to manually downgrade to 12.4.

**Comment by lynnmunday** [comment]
Is there an option when you upgrade to Big Sur not to upgrade the Xcode command line tools?

**Reply by milljm** [reply]
I am not aware that an upgrade to Big Sur upgrades XCode as well. Usually quite the opposite in my experience. I usually run into folks who upgraded their OS, but then fail to upgrade their XCode.

Edit: Oh I see the issue. Perhaps they 'fixed' that by lumping both together. Traditionally you're supposed to update both. But in this case, yes... we need to stay away from 12.5 for the time being.

If you wish to upgrade to Big Sur, you should be able to do that via the App Store (search for Big Sur, and install).

**Comment by cticenhour** [comment]
MOOSE now supports XCode 12.5, 13, and 13.1! See #19289 for more details and information about how to update.
